lib/improver/tests/utilities/test_temporal.py

```diff
@@ -77,6 +78,27 @@ def test_define_cycletime_format(self):
         self.assertEqual(result, dt)
 
 
+class Test_datetime_to_cycletime(IrisTest):
+
+    """Test that a datetime object can be converted into a cycletime
+    of a format such as YYYYMMDDTHHMMZ."""
+
+    def test_basic(self):
+        """Test that a datetime object is returned of the expected value."""
+        dt = datetime.datetime(2017, 11, 22, 1, 0)
+        cycletime = "20171122T0100Z"
+        result = datetime_to_cycletime(dt)
+        self.assertIsInstance(result, str)
+        self.assertEqual(result, cycletime)
+
+    def test_define_cycletime_format(self):
+        """Test when a cycletime is defined."""
+        dt = datetime.datetime(2017, 11, 22, 1, 0)
+        cycletime = "201711220100"
+        result = datetime_to_cycletime(dt, cycletime_format="%Y%m%d%H%M")
+        self.assertEqual(result, cycletime)
+
```

Comment:
Could you stick in a test demonstrating that it handles seconds (e.g. by ignoring them unless a custom format is included that uses them).

Reply:
I've added a unit test to demonstrate the handling of seconds.